src/app/shared/routing/group-router.ts

```diff
@@ -15,7 +15,7 @@ export class GroupRouter {
    * Navigate to given group, on the path page.
    * If page is not given and we are currently on a group page, use the same page. Otherwise, default to 'details'.
    */
-  navigateTo(route: GroupRoute, page?: 'edit'|'details'): void {
+  navigateTo(route: GroupRoute, page?: 'edit'|'details'|'users'): void {
```

Comment:
The same everywhere here, I don't think that's a good idea to put it as the same level as 'edit' and 'details' as it is not.
For the navigation just navigate "manually" to "[ '/', 'groups', 'users', id ]" for now, out of this router. I'll see later how/if we integrate users in this group router (if they need route as well).

Reply:
Fixed